tests/faker.py

```diff
@@ -0,0 +1,23 @@
+def fake_creds():
+    return {
+        'FAKE_CHANNEL': 'C12942JF92'
+    }
```

Comment:
I don't think making it global prevents it from being used anywhere in others test cases. I believe it more or less a constant - so i don't expect its state to be changed anywhere  it's used.